doc/installation.rst

```diff
@@ -84,8 +95,8 @@ hold.
 
 .. _generic_install:
 
-Generic SfePy Installation Instructions
----------------------------------------
+Installing SfePy from sources
```

Comment:
sources -> Sources
Note that this change needs to be reflected in links like `Generic SfePy Installation Instructions`_...